dap-mode.el

```diff
@@ -1399,10 +1402,10 @@ before starting the debug process."
          (dap--initialize-message type)
          (dap--session-init-resp-handler
           debug-session
-          (lambda (initialize-result)
+          (lambda (current-capabilities)
```

Comment:
You can use `-lambda` here to bind to capabilities directly without additional `(ht-get current-capabilities "body")` later